CMakeLists.txt

```diff
@@ -365,8 +368,22 @@ if (MSVC OR XCODE)
   set(LIT_ARGS_DEFAULT "${LIT_ARGS_DEFAULT} --no-progress-bar")
 endif()
 
-# HLSL Change - disable short output mode for tests (will list all tests)
-set(LLVM_LIT_ARGS "-v --no-progress-bar" CACHE STRING "Options for lit")
+# HLSL Change start - set arguments for the lit runner
+option(HLSL_TESTS_VERBOSE_OUTPUT "Enable verbose test output" OFF) # HLSL Change
```

Comment:
Is there a reason for doing these as CMake options instead of just doing them as `hctbuild` options? If so, could we alternatively make them options implemented in a cache file like PredefinedParams.cmake?

The concern that I have here is that you’re adding new options that have values which are cached, and those in turn contribute to the default cached value of another option. CMake has no strategy for cache invalidation, so cached options defaulting based on other cached options results in really odd behavior usually forcing users to throw up their arms in disgust, delete their build directories and start all over again.